libs/output-mapping/tests/Writer/File/Strategy/ABSWorkspaceTest.php

```diff
@@ -82,32 +82,67 @@ public function testCreateStrategyInvalidWorkspace()
         self::assertSame([], $files);
     }
 
-    public function testListFilesWorkspaceDropped()
+    public function testListFilesWorkspaceDropped(): void
     {
-        $strategy = new ABSWorkspace($this->clientWrapper, new TestLogger(), $this->getProvider(), $this->getProvider(), 'json');
+        $strategy = new ABSWorkspace(
+            $this->clientWrapper,
+            new TestLogger(),
+            $this->getProvider(),
+            $this->getProvider(),
+            'json'
+        );
         $workspaces = new Workspaces($this->clientWrapper->getBasicClient());
         $workspaces->deleteWorkspace($this->workspace['id']);
         self::expectException(InvalidOutputException::class);
         self::expectExceptionMessage('Failed to list files: "The specified container does not exist.".');
         $strategy->listFiles('data/out/files');
     }
 
-    public function testListFiles()
+    public function testListFiles(): void
     {
-        $strategy = new ABSWorkspace($this->clientWrapper, new TestLogger(), $this->getProvider(), $this->getProvider(), 'json');
-        $blobClient = ClientFactory::createClientFromConnectionString($this->workspace['connection']['connectionString']);
-        $blobClient->createAppendBlob($this->workspace['connection']['container'], 'data/out/files/my-file');
-        $blobClient->createAppendBlob($this->workspace['connection']['container'], 'data/out/files/my-file.manifest');
-        $blobClient->createAppendBlob($this->workspace['connection']['container'], 'data/out/files/my-second-file');
-        $blobClient->createAppendBlob($this->workspace['connection']['container'], 'data/out/files/my-second-file.manifest');
-        $blobClient->createAppendBlob($this->workspace['connection']['container'], 'data/out/tables/my-other-file');
+        $strategy = new ABSWorkspace(
+            $this->clientWrapper,
+            new TestLogger(),
+            $this->getProvider(),
+            $this->getProvider(),
+            'json'
+        );
+        $blobClient = ClientFactory::createClientFromConnectionString(
+            $this->workspace['connection']['connectionString']
+        );
+        $blobClient->createAppendBlob(
+            $this->workspace['connection']['container'],
+            'data/out/files/my-file'
+        );
+        $blobClient->createAppendBlob(
+            $this->workspace['connection']['container'],
+            'data/out/files/my-file.manifest'
+        );
+        $blobClient->createAppendBlob(
+            $this->workspace['connection']['container'],
+            'data/out/files/my-second-file'
+        );
+        $blobClient->createAppendBlob(
+            $this->workspace['connection']['container'],
+            'data/out/files/my-second-file.manifest'
+        );
+        $blobClient->createAppendBlob(
+            $this->workspace['connection']['container'],
+            'data/out/tables/my-other-file'
+        );
```

Comment:
Ja bych to vynuceny zalamovani zrusil.. at me nekdo zkusi presvedcit, ze takle je to citelnejsi :smile:

Reply:
🤷‍♂️ 
buď rád, že to není python